scripts/gha/github.py

```diff
@@ -0,0 +1,280 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A utility for GitHub REST API.
+
+This script handles GitHub Issue, Pull Request, Comment, Label and Artifact
```

Comment:
NIT: the name is too general, maybe call github_rest_utils.py?

Reply:
Thanks for the suggestion. 

Yes, we have a few similar tools in both CPP and Unity repo with subtle difference. 
I'm considering combining and moving these tools in a new repo (e.g. firebase-games-testing) in the future. But, right now the highest priority it's to make it work in Unity repo without breaking CPP CI.

Yes. I create [set_repo_url()](https://github.com/firebase/firebase-cpp-sdk/pull/746/files#diff-bd5d647c8c782da640a271074167ff54d565487e435358f3c5972c7ddea7e9c8R43) function in the script before I create this PR. 

I also don't see a big change of this files in the near future.